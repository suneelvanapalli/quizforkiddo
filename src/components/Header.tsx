export default function Header() {
  return <p data-testid='header'>test</p>;
}
