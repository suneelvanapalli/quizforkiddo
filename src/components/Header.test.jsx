import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import Header from './Header';

describe('Header', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
});
