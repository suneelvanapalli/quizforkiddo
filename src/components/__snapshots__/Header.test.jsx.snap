// Vitest Snapshot v1

exports[`Header > should match snapshot 1`] = `
<body>
  <div>
    <p
      data-testid="header"
    >
      test
    </p>
  </div>
</body>
`;

exports[`Header > should render a paragraph 1`] = `
<body>
  <div>
    <p
      data-testid="header"
    >
      test
    </p>
  </div>
</body>
`;
